web/Dockerfile

```diff
@@ -1,14 +1,15 @@
 # web
 FROM fbg-common:latest AS common
-FROM node:14.5.0-buster
-RUN groupadd -g 999 appuser && useradd -m -d /appdata -r -u 999 -g appuser appuser
+FROM node:14.15.0-alpine3.12 AS builder
+RUN apk add --no-cache autoconf automake file g++ libtool make nasm libpng-dev libc6-compat
+RUN addgroup -g 938 appuser && adduser -h /appdata -S -u 997 -G appuser appuser
 RUN rm /bin/su
```

Comment:
just curious if this is better stated ```RUN rm -rf /bin/su```  ? I'm not sure if disk leaks are possible but this looks like it would create one.

Reply:
@larry801 ignore comment about placing directives above FROM in docker file I was misreading this from the commit history shown on review.  That is also probably true of my question about dependency being moved in package.json

Reply:
@larry801 similarly, my comments about rm /bin/su were a misunderstanding/perception that this was a directory like /usr/sbin brain is on some sort of time delay today... this looks fine.